examples/pxScene2d/src/initGL.js

```diff
@@ -650,6 +671,65 @@ async function loadMjs(source, url, context)
     };
   };
 
+  function loadESM(filename) {
```

Comment:
Please revert. For some reason you removed my changes on bootstrap. Please merge everything from _webgl to your branch.